.github/workflows/flutter.yaml

```
@@ -0,0 +1,35 @@
+
+      - name: Install dependencies
+        run: flutter pub get
+
+      - name: Dart analyze
+        run: flutter analyze
+
+      - name: Check formatting
+        run: dart format --output=none --set-exit-if-changed .
+
+      - name: Flutter build (apk)
+        run:  flutter build apk --release --no-tree-shake-icons | grep -v "deprecated"
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)